horovod/spark/lightning/estimator.py

```diff
@@ -170,7 +170,13 @@ class TorchEstimator(HorovodEstimator, TorchEstimatorParamsWritable,
     logger = Param(Params._dummy(), 'logger', 'optional logger')
 
     log_every_n_steps = Param(Params._dummy(), 'log_every_n_steps', 'control the frequency of logging',
-                                typeConverter=TypeConverters.toInt)
+                              typeConverter=TypeConverters.toInt)
+
+    data_loader_class = Param(Params._dummy(), 'data_loader_class',
+                              'Name of the dataloader class.')
+
+    loader_num_epochs = Param(Params._dummy(), 'loader_num_epochs',
```

Comment:
When would the user want to set this? It seems we would always want to set it to `None` and let the trainer decide when stop reading.